src/posix.mak

```diff
@@ -237,11 +237,12 @@ else
     FRONT_SRCS += libelf.d scanelf.d
 endif
 
-GLUE_SRCS=$(addsuffix .d, irstate toelfdebug toctype gluelayer todt tocsym toir)
+GLUE_SRCS=$(addsuffix .d, irstate toelfdebug toctype glue gluelayer todt tocsym toir dmsc \
```

Comment:
This is why it doesn't build.  There is no glue.d in this branch.  Most of the changes in this file should be removed.